app/CMakeLists.txt

```diff
@@ -4,54 +4,69 @@
 #-----------------------------------------------------------------------------#
 
 if(BUILD_TESTING)
-  if(CELERITAS_USE_Python)
-    set(CELER_PYTHON "$<TARGET_FILE:Python::Interpreter>")
-    set(CELER_NEEDS_PYTHON)
-  else()
-    set(CELER_PYTHON "python3")
-    set(CELER_NEEDS_PYTHON DISABLED true)
-  endif()
+    if(CELERITAS_USE_Python)
```

Comment:
Do we mean to intent by 4?

Reply:
Yep thank you, I've added it to the rc file.